### PR TITLE
Docs for Manual Refresh

### PR DESCRIPTION
**Pull request by Jeadie**
## 🗣 Description
 - Docs for changes to manual refresh
 - https://github.com/spiceai/spiceai/pull/2515

### CODE REVIEW

File: spiceaidocs/docs/api/http/refresh.md [hunk 1]
```diff
@@ -9,10 +9,19 @@ pagination_next: null
...
+ - `refresh_mode` (String, Optional): Refresh mode to use, see [Refresh Modes docs](/components/data-accelerators/data-refresh.md#refresh-modes). Defaults to `refresh_mode` specified in the spicepod.
+
+ Example:
 
 ```bash
-curl -i -XPOST 127.0.0.1:8090/v1/datasets/taxi_trips/acceleration/refresh
+curl -i -XPOST 127.0.0.1:8090/v1/datasets/taxi_trips/acceleration/refresh \
+  -H "Content-Type: application/json" \
+  -d '{
+    "refresh_sql": "SELECT * FROM taxi_trips WHERE trip_amount > 10.0",
```
Comment (Sevenannn) [inline spiceaidocs/docs/api/http/refresh.md]: "refresh_sql": "SELECT * FROM taxi_trips WHERE tip_amount > 10.0",

File: spiceaidocs/docs/api/http/refresh.md [hunk 2]
```diff
@@ -9,10 +9,19 @@ pagination_next: null
 
 Performs an on-demand refresh for an accelerated dataset. On-demand refresh applies only to `full` and `append` refresh modes (not `changes`).
 
-Example:
+Request Body:
+ - `refresh_sql` (String, Optional): Refresh SQL to use, see [Refresh Sql docs](/components/data-accelerators/data-refresh.md#refresh-sql). Defaults to `refresh_sql` specified in the spicepod.
```
Comment (phillipleblanc) [inline spiceaidocs/docs/api/http/refresh.md]:
```suggestion
 - `refresh_sql` (String, Optional): Refresh SQL to use, see [Refresh SQL docs](/components/data-accelerators/data-refresh.md#refresh-sql). Defaults to the `refresh_sql` specified in the spicepod.
```

File: spiceaidocs/docs/cli/reference/refresh.md [hunk 3]
```diff
@@ -18,12 +18,15 @@ spice refresh [dataset] [flags]
 #### Flags
 
 - `--tls-root-certificate-file`   The path to the root certificate file used to verify the Spice.ai runtime server certificate
+- `--refresh-sql`  SQL used to refresh the dataset, see [Refresh Sql docs](/components/data-accelerators/data-refresh.md#refresh-sql).
```
Comment (phillipleblanc) [inline spiceaidocs/docs/cli/reference/refresh.md]:
```suggestion
- `--refresh-sql`  SQL used to refresh the dataset, see [Refresh SQL docs](/components/data-accelerators/data-refresh.md#refresh-sql).
```

**Review by phillipleblanc — DISMISSED** [review]

File: spiceaidocs/docs/api/http/refresh.md (hunk 1 above)
Comment (Jeadie) [inline spiceaidocs/docs/api/http/refresh.md]:
```suggestion
    "refresh_sql": "SELECT * FROM taxi_trips WHERE tip_amount > 10.0",
```

**Review by Sevenannn — APPROVED** [review]

**Pull request merged**